django/db/backends/base/operations.py

```diff
@@ -231,6 +231,13 @@ def field_cast_sql(self, db_type, internal_type):
         )
         return "%s"
 
+    def force_group_by(self):
```

Comment:
I would use `force_no_grouping` to more consistent with `force_no_ordering`:
```suggestion
    def force_no_grouping(self):
```

Reply:
`force_no_odering()` is to literally force "no order" right? eg `ORDER BY NULL` in MySQL.  Here we're forcing a group by (we're reacting to a "no grouping" and forcing a "group by").

I just don't think `force_no_grouping()` makes sense 🤷‍♂️

Reply:
True :+1:

Reply:
where's kezabelle when you need a linguistic expert lol